### PR TITLE
[TIMOB-9366] update radial backgroundGradient example to match implementation

### PR DESCRIPTION
**Pull request by Topener**
JIRA: https://jira.appcelerator.org/browse/TIMOB-9366

The current example in the documentation works only for iOS. I updated it to reflect both platforms

### CODE REVIEW

File: apidoc/Titanium/UI/View.yml [hunk 1]
```diff
@@ -1119,7 +1119,7 @@ properties:
                         startPoint: { x: 50, y: 50 },
                         endPoint: { x: 50, y: 50 },
                         colors: [ 'red', 'blue'],
-                        startRadius: '90%',
+                        startRadius: 90,
```
Comment (jquick-axway) [inline apidoc/Titanium/UI/View.yml]: We should set startRadius to `50` instead.
This way, a full circle will just-fit within the view.

Note that setting this to `90` would make the radius larger than the old `90%` setting. On iOS, `100%` equals `sqrt(width^2, height^2) / 2`, which is the distance between the center of the view to one of its corners.

**Review by jquick-axway — CHANGES_REQUESTED** [review]

**Comment by Topener** [comment]
@jquick-axway good point! Updated, will update the other PR too

**Review by jquick-axway — APPROVED** [review]
CR: Pass

**Comment by lokeshchdhry** [comment]
FR Passed.

Example works properly on both Android & IOS.

Studio Ver:     5.0.0.201712081732
SDK Ver:        7.2.0 local build
OS Ver:         10.13.2
Xcode Ver:      Xcode 9.2
Appc NPM:       4.2.12
Appc CLI:       7.0.2
Daemon Ver:     1.0.1
Ti CLI Ver:     5.0.14
Alloy Ver:      1.11.0
Node Ver:       8.9.1
NPM Ver:        5.5.1
Java Ver:       1.8.0_101
Devices:        ⇨ google Nexus 5 --- Android 6.0.1
                ⇨ google Nexus 6P --- Android 8.0.0

**Pull request merged**